datascience/README.md

```diff
@@ -52,3 +59,41 @@ Returns :
 3      DE      4229.0  biomass  2016-12-23T00:00:00+00:00
 4      DE     19919.0     coal  2016-12-23T00:00:00+00:00
 ```
+
+#### get_exchange(countries, start_date, end_date)
```

Comment:
Exchanges are expressed and stored in form of power, so instaneous value. There's no aggregation done (the `server.js` link you gave just fetches the last value of each exchange). It's exactly the same as in the production endpoint.